client/client.go

```diff
@@ -146,6 +146,11 @@ func NewClient(baseDir string) (*Client, error) {
 		return nil, fmt.Errorf("error reading CT file: %v", err)
 	}
 
+	// Create the initial sync file.
+	if err := c.updateSyncFile(); err != nil {
+		panic("could not create sync file: " + fmt.Sprintf("%v", err))
```

Comment:
this should be a `return nil, fmt.Errorf()` like all of the other startup errors in this function. The decision about what to do if startup fails should be made by the caller